packages/client-react/src/translations.js

```diff
@@ -2,10 +2,14 @@ const translations = {
   en: {
     'common.Toolbar.moveForward': 'Move forward',
     'common.Toolbar.moveBack': 'Move back',
+    'nothingToShow': 'Nothing to show',
```

Comment:
Please, rename the key to 'common.NoFilesFoundStub.title' according to convention in the issue https://opuscapita.atlassian.net/browse/EPROC-20549